gptcli/providers/openai.py

```diff
@@ -15,10 +15,16 @@
     UsageEvent,
 )
 
+use_azure: bool = False
 
 class OpenAICompletionProvider(CompletionProvider):
```

Comment:
probably would want `class AzureOpenAICompletionProvider(OpenAICompletionProvider):` that overrides the client